draft-ietf-quic-tls.md

```diff
@@ -804,17 +803,14 @@ modifying the contents of packets from future versions.
 The HKDF-Expand-Label function defined in TLS 1.3 MUST be used for Initial
 packets even where the TLS versions offered do not include TLS 1.3.
 
+The secrets used for protecting Initial packets do not change during the
+connection, even after receiving a Retry.  A server that sends a Retry
+therefore needs to either remember Initial protection keys or save them
```

Comment:
Shouldn't this just say that the server already needs to have access to the original destination CID because of the `original_connection_id` transport parameter? I don't think the server needs to remember/encode initial keys on top of that

Reply:
Good point, MT suggested some simplifications to the text and I added a sentence clarifying that this is not a new requirement, it just changes when the connection ID is used.